modules/RTC/JitsiRemoteTrack.js

```diff
@@ -12,23 +12,30 @@ var ttfmTrackerVideoAttached = false;
 
 /**
  * Represents a single media track (either audio or video).
- * @param rtc {RTC} the RTC service instance.
- * @param ownerJid the MUC JID of the track owner
- * @param stream WebRTC MediaStream, parent of the track
- * @param track underlying WebRTC MediaStreamTrack for new JitsiRemoteTrack
- * @param mediaType the MediaType of the JitsiRemoteTrack
- * @param videoType the VideoType of the JitsiRemoteTrack
- * @param ssrc the SSRC number of the Media Stream
- * @param muted initial muted state of the JitsiRemoteTrack
+ * @param {RTC} rtc the RTC service instance.
+ * @param {JitsiConference} conference the conference to which this track
+ *        belongs to
+ * @param {string} owner the endpoint ID of the track owner
+ * @param {MediaStream} stream WebRTC MediaStream, parent of the track
+ * @param {MediaStreamTrack} track underlying WebRTC MediaStreamTrack for
+ *        the new JitsiRemoteTrack
+ * @param {MediaType} mediaType the type of the media
+ * @param {VideoType} videoType the type of the video if applicable
+ * @param {string} ssrc the SSRC number of the Media Stream
+ * @param {boolean} muted the initial muted state
+ * @param {boolean} isP2P indicates whether or not this track belongs to a P2P
```

Comment:
yes, the can have the same "msid", but will be "spawned" by different PeerConnection and will be a different JitsiRemoteTrack instance.